pages/linux/amap.md

```diff
@@ -0,0 +1,20 @@
+# amap
+
+> Next-generation scanning tool for pentesters.
+> More information: <https://www.thc.org/>.
+
+- Scan port 80 on target IP address and print ASCII banner response, omit closed ports, verbosely:
+
+`amap -bvq {{ip_or_hostname}} {{80}}`
+
+- Scan port 1-1024 and capture banners:
+
+`amap -bvq {{ip_or_hostname}} {{1-1024}} -B`
```

Comment:
We prefer long arguments if available instead of `-B`

Reply:
Unfortunately application does not support long arguments